plugins/modules/dcnm_policy.py

```diff
@@ -338,6 +349,46 @@
       - name: POLICY-103103
       - switch:
           - ip: "{{ ansible_switch1 }}"
+
+# Use the description as key
+
+# NOTE: As the description of the policy in NDFC/DCNM is not unique,
+#       the user must make sure no policies with the same description are created on NDFC out of the playbook.
+#       If the description is not unique, the module will raise an error.
+
+## Below task will policies with description "policy_radius" on swtich1, switch2 and switch3,
```

Comment:
typo, should read as

Below task will create policies ...

Reply:
will fix it.